From 80628e402a48461d88b984ae0e2db2f567a611c5 Mon Sep 17 00:00:00 2001
From: Alex Elder <elder@inktank.com>
Date: Fri, 1 Mar 2013 18:00:15 -0600
Subject: [PATCH 672/938] libceph: kill ceph_msg->pagelist_count
References: fate#312983
Git-commit: ec02a2f2ffae13e038453ae89592a8c6210f7f4d (partial)
Patch-mainline: v3.10

The pagelist_count field is never actually used, so get rid of it.

Signed-off-by: Alex Elder <elder@inktank.com>
Reviewed-by: Josh Durgin <josh.durgin@inktank.com>
Acked-by: Danny Al-Gaaf <dalgaaf@suse.de>

---
 net/ceph/messenger.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/ceph/messenger.c b/net/ceph/messenger.c
index 83b42e6..3418724 100644
--- a/net/ceph/messenger.c
+++ b/net/ceph/messenger.c
@@ -2713,7 +2713,6 @@ struct ceph_msg *ceph_msg_new(int type, int front_len, gfp_t flags,
 	m->page_count = 0;
 	m->page_alignment = 0;
 	m->pages = NULL;
-	m->pagelist_count = 0;
 	m->pagelist = NULL;
 #ifdef	CONFIG_BLOCK
 	m->bio = NULL;
@@ -2893,7 +2892,6 @@ void ceph_msg_last_put(struct kref *kref)
 		ceph_pagelist_release(m->pagelist);
 		kfree(m->pagelist);
 		m->pagelist = NULL;
-		m->pagelist_count = 0;
 	}
 
 	m->trail = NULL;
-- 
1.8.3

